Core/Inc/FreeRTOSConfig.h

```diff
@@ -69,13 +69,15 @@
 #define configTICK_RATE_HZ                       ((TickType_t)1000)
 #define configMAX_PRIORITIES                     ( 56 )
 #define configMINIMAL_STACK_SIZE                 ((uint16_t)128)
-#define configTOTAL_HEAP_SIZE                    ((size_t)15360)
+#define configTOTAL_HEAP_SIZE                    ((size_t)20000)

```

Comment:
I thought I was running out of heap and I think it solved an issue after I increased the heap size. How much heap we use is not a concern at the moment since we have 192 kB of RAM on the stm.